### PR TITLE
Hopper.com

### PR DESCRIPTION
**Pull request by Mlg29**
### What does the PR do?
-Work on the homepage ie (home.html)
-Work on the company section ie company.html (The upper part)
-Work on the people section ie people.html (From making the world)
-Work on the term of services page ie termofservice.html
-All pages are desktop and mobile responsive

### Any relevant info?
-Didn't make use of the reusable component for now(waiting for the team lead to push the reusable component)

### Screenshot
cm
comp
Homebody
homek
Homepage
mob
mobile
pp
ppp
tt

### CODE REVIEW

File: components/header.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <body>
     <header id="home">
         <nav class="navbar">
-        <div class="container">
+        <div class="containe">
```
Comment (SEUNAGBEYE) [inline components/header.html]: do you mean `container`?

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]

File: components/header.html (hunk 1 above)
Comment (Mlg29) [reply components/header.html]: The container zainab used was conflicting with mine, so i removed the r pending the time zainab will work on the new header

Comment (Mlg29) [reply components/header.html]: I used a bootstrap container, and zainab also had a container styled for the header.
Though she said she had resolved it. just waiting for it to be merged and i change my header

**Comment by SEUNAGBEYE** [comment]
Heya,

Good job, just a few more tweaks left

1. This was supposed to the a carousel
Screen Shot 2019-09-25 at 10 54 50 AM

2. The style for this part of the screen is missing
Screen Shot 2019-09-25 at 10 56 41 AM

3. Ensure both screens are responsive

**Comment by Mlg29** [comment]
Good morning Seun.
 For the image, i know its meant to be a carousel but didnt know of the
image to add to it to make the carousel work. i was only given 1 image for
that particular place in the mockup.

Also about the other part of the section. it wasnt my task.

**Comment by SEUNAGBEYE** [comment]
I believe this is obsolete

**Pull request closed without merging**